src/string_editor_window.cpp

```diff
@@ -42,7 +42,7 @@ std::pair<int, int> string_editor_window::get_line_and_position( std::vector<std
         utf8_wrapper linetext( foldedtext[i] );
         int temp = linetext.display_width();
         //foldstring, cuts " " away, so it is possible to get a hughe disconect between folded and unfolded string.
-        temp += ( foldedtext[i].back() != ' ' ) ? 1 : 0;
+        if(!foldedtext[i].empty()) temp += ( foldedtext[i].back() != ' ' ) ? 1 : 0;
```

Comment:
```suggestion
        if( !foldedtext[i].empty() ) temp += ( foldedtext[i].back() != ' ' ) ? 1 : 0;
```
I think these are the only changes that's needed (easier than trying to understand astyle).